lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -219,6 +220,39 @@
                 },
 
                 /**
+                 * Render the edit team form.
+                 */
+                editTeam: function (topicID, teamID) {
+                    var self = this;
+                    this.getTeamsView(topicID).done(function (teamsView) {
+                        self.getTeam(teamID, false).done(function(team) {
+                            self.mainView = new ViewWithHeader({
```

Comment:
This code is very difficult to read and understand. Perhaps pull some of it out into variables so they can be named? At the very least, put a comment in about what "false" signifies on line 228.